src/LibraryManager.Vsix/Json/Completion/CompletionController.cs

```diff
@@ -122,7 +122,7 @@ private void OnCommitted(object sender, EventArgs e)
                 return;
             }
 
-            var doc = JSONEditorDocument.FromTextView(_textView);
+            var doc = JSONEditorDocument.FromTextBuffer(_textView.TextBuffer);
```

Comment:
`FromTextView` is implemented by using `textView.TextDataModel.DocumentBuffer`.  Maybe we should do the same here?  I think `_textView.TextBuffer` is not always the same content as the document buffer (maybe for diff?).